### PR TITLE
[Bridge\PhpUnit] Fix composer installed phpunit detection

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #14849 |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: src/Symfony/Bridge/PhpUnit/bootstrap.php [hunk 1]
```diff
@@ -4,7 +4,7 @@
 use Symfony\Bridge\PhpUnit\DeprecationErrorHandler;
 
 // Detect if we're loaded by an actual run of phpunit
-if (!class_exists('PHPUnit_TextUI_Command', false)) {
+if (!defined('PHPUNIT_COMPOSER_INSTALL') && !class_exists('PHPUnit_TextUI_Command', false)) {
```
Comment (xabbuh) [inline src/Symfony/Bridge/PhpUnit/bootstrap.php]: Where is this constant defined and is it guaranteed that it won't be removed in the future?

Comment (GromNaN) [inline src/Symfony/Bridge/PhpUnit/bootstrap.php]: Here : https://github.com/sebastianbergmann/phpunit/blob/master/phpunit#L18

**Comment by stof** [comment]
:+1:

**Comment by Tobion** [comment]
:+1:

**Comment by xabbuh** [comment]
:+1:

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**